### PR TITLE
Adds identification card compartmentalizer

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds the identification card compartmentalizer, works similarly to the disk compartmentalizer, but for ids and pdas!
## Why It's Good For The Game

Its good to have a place for storing lost and terminated ID/PDAs. Currently (specially on highpop) the HoP office can become a mess full of ids laying around without a proper destination, this PR adds a destination.
https://www.paradisestation.org/forum/topic/19510-hop-box-for-lostterminated-idspdas/#comment-149037
## Images of changes

idbox
meta
delta

## Testing

- Tried to insert id types and non id types (wich got refused), working properly.
## Changelog
:cl:
add: Added the identification card compartmentalizer
imageadd: Added the identification card compartmentalizer sprite
/:cl:

### CODE REVIEW

**Comment by procdrone** [comment]
absolutely amazing!

File: code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm [hunk 1]
```diff
@@ -663,8 +663,8 @@
 
 /**
   * # Disk Compartmentalizer
-  *
-  * Disk variant of the [Smart Fridge][/obj/machinery/smartfridge].
+  * # Identification Compartmentalizer
+  * Variants of the [Smart Fridge][/obj/machinery/smartfridge].
   */
 /obj/machinery/smartfridge/disks
```
Comment (AffectedArc07) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm:669]: This is an autodoc comment and is specific to that type. Revert this change.

Comment (Henri215) [reply code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: Didn't know it was an autodoc comment, it has been reverted.

**Review by SteelSlayer — APPROVED** [review]
